src/marks/index.js

```diff
@@ -74,10 +77,69 @@ class Strike extends TipTapStrike {
 
 }
 
+class Link extends TipTapLink {
+
+	get schema() {
+		const inherited = super.schema
+		return {
+			...inherited,
+			parseDOM: [
+				{
+					tag: 'a[href]',
+					getAttrs: dom => ({
+						href: parseHref(dom),
+					}),
+				},
+			],
+			toDOM: node => ['a', {
```

Comment:
When running into a conflict during rebase you mainly need the title to be added here:
https://github.com/nextcloud/text/pull/878/files#r440731486

Reply:
thanks for the heads up. Did just that. :heavy_check_mark: